### PR TITLE
'Expand on image hover' option

### PR DESCRIPTION
**Pull request by discomrade**
Fix and enable image hover, an option for having images expand when you hover over them, similar to GETchan's default behavior.
This is currently not enabled for the user by default.  They must turn it on in the options menu.

### CODE REVIEW

File: js/image-hover.js [hunk 1]
```diff
@@ -4,22 +4,20 @@
...
-
+$(document).ready(function () {
 if (window.Options && Options.get_tab('general')) {
 	Options.extend_tab("general", 
 	"<fieldset><legend>Image hover</legend>"
 	+ ("<label class='image-hover' id='imageHover'><input type='checkbox' /> "+_('Image hover')+"</label>")
 	+ ("<label class='image-hover' id='catalogImageHover'><input type='checkbox' /> "+_('Image hover on catalog')+"</label>")
 	+ ("<label class='image-hover' id='imageHoverFollowCursor'><input type='checkbox' /> "+_('Image hover should follow cursor')+"</label>")
-	+ "</fieldset>");
+	+ "</fieldset>");console.log("aaa");
```
Comment (nonmakina) [inline js/image-hover.js]: stale debug

File: js/image-hover.js [hunk 2]
```diff
@@ -71,7 +70,7 @@ function initImageHover() { //Pashe, influenced by tux, et al, WTFPL
 	if (!getSetting("imageHover") && !getSetting("catalogImageHover")) {return;}
 	
 	var selectors = [];
-	
+		console.log("image-hover");
```
Comment (nonmakina) [inline js/image-hover.js]: stale debug

File: js/image-hover.js [hunk 3]
```diff
@@ -96,38 +95,41 @@ function initImageHover() { //Pashe, influenced by tux, et al, WTFPL
...
 
+function followCursor(e, hoverImage) {
+	var scrollTop = $(window).scrollTop();
+	var imgWidth = Number(hoverImage.css("max-width").slice(0,-2))
+	var imgHeight = Number(hoverImage.css("max-height").slice(0,-2))
+	var imgTop = e.pageY - (imgHeight/2);
+	var windowWidth = $(window).width();
+	var imgEnd = imgWidth + e.pageX;
+	
+	if (imgTop < scrollTop + 15) {
```
Comment (nonmakina) [inline js/image-hover.js:106]: nice

**Review by nonmakina — DISMISSED** [review]

**Review by towards-a-new-leftypol — APPROVED** [review]
The changes themselves look good, my only concern going forward is that the front-end is getting way bloated with scripts. We should be picky about which scripts to enable, maybe even have telemetry to tell us which addons users actually use...

anyway approved.

**Pull request merged**